src/components/Header/Header.tsx

```diff
@@ -1,14 +1,11 @@
 import React from "react";
-import logo from '../../media/ok200logo.jpeg';
-import './Header.scss';
+import "./Header.scss";
+import logo from "../../media/ok200logo.jpeg";
 
-
-const Header = () => {
-    return (
-        <div className="header__container">
-            <img className="logo" src={logo} alt="ok200-logo" />
-        </div>
-    )
-}
-
-export default Header;
\ No newline at end of file
+export const Header: React.FC = () => {
```

Comment:
For this export syntax, you should tac on `default`, because React components should only ever export one function (the component function). So like this:
```
export default const Header: React.FC = () => {
```